### DISCUSSION TITLE
New discussion category items have incident alignment

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#43960] [Mobile] [answered]
I'd expect each icon to have the same width or be padded to the same width such that each item's label lines up with all the other labels.

![Screenshot_20230111-091300.png](https://user-images.githubusercontent.com/2119212/211831260-535dd230-f9bc-49b8-9c29-38688bfd4505.png)

**Comment by jacortinas** [comment, marked as answer]
Hi, @jsoref! I've created an issue to track this. Thank you!
